packages/dx-react-grid/docs/reference/grid-exporter.md

```diff
@@ -0,0 +1,41 @@
+# GridExporter Component Reference
+
+A component that performs data export to Excel format.
+
+## Import
+
+Use the following statement to import the component:
+
+```js
+import { GridExporter } from '@devexpress/dx-react-grid';
+```
+
+## User Reference
+
+### Dependencies
+
+None.
+
+### Properties
+
+Name | Type | Default | Description
+-----|------|---------|------------
+rows | ReadonlyArray&lt;any&gt; | | An array containing custom data. A user defines the access to this data. Refer to [Data Accessors](../guides/data-accessors.md) for details.
+columns | ReadonlyArray&lt;[Column](#column)&gt; | | Specifies for which row fields columns are created.
+getRowId? | (row: any) => number &#124; string | | Specifies the function used to get a unique row identifier. Define this function if the identifier is different than the row index.
+getCellValue? | (row: any, columnName: string) => any | | Specifies the function used to get a cell's value.
+filters? | Array&lt;[Filter](#filter)&gt; | | Specifies the applied filters.
+defaultFilters? | Array&lt;[Filter](#filter)&gt; | [] | Specifies the filters initially applied in the uncontrolled mode.
+sorting? | Array&lt;[Sorting](#sorting)&gt; | | Specifies the applied sorting.
+grouping? | Array&lt;[Grouping](#grouping)&gt; | | Specifies columns to group by.
+groupColumnExtensions? | Array&lt;[GroupingState.ColumnExtension](#groupingstatecolumnextension)&gt; | | Additional column properties.
+showColumnsWhenGrouped? | boolean | false | A Boolean value that specifies whether the grid's table displays a column by which data is grouped.
```

Comment:
It seems that we should rename this prop because from reading it I get an impression that it affects ALL columns: whether we show all of the columns or not when we use grouping. But it affects only those the Grid's data is grouped by. So, I think, something like `showGroupingColumns` would be better

Reply:
The name comes from the TableGroupRow plugin